docs/rfcs/rfc-009-graphql-ui-poc.md

```diff
@@ -0,0 +1,112 @@
+# GraphQL Graph App PoC
+
+## Problem
+
+The [GraphQL Architect](https://grandstack.io/docs/graphql-architect-overview/#:~:text=GraphQL%20Architect%20is%20a%20graph,Neo4j%20Desktop%20Graph%20Apps%20Gallery) for usage in the Neo4j Desktop is due to be replaced. As an initial step a proof of concept of a successor shall be established.
+
+## Proposed Solution
+
+_Name suggestions:_  
+Explorer  
```

Comment:
I plus +1 'Explorer'

Reply:
I also side with Explorer for the most part, apart from the fact that this application will also let you _edit_ the type definitions, which for me goes beyond "exploring"!

Reply:
Yes I came to like the name as well yet sadly it's already taken (sorry that's one me!), https://www.apollographql.com/docs/studio/explorer/explorer/ 
We may have to go with a working name for this first iteration/PoC.